app/src/main/java/com/example/bigowlapp/activity/BigOwlActivity.java

```diff
@@ -17,12 +19,15 @@
     ImageButton imgBtnUser;
     ImageButton imgBtnNotification;
     ImageButton imgBtnBigOwl;
+    private ProgressBar progressBar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(getContentView());
 
+        progressBar = (ProgressBar) findViewById(R.id.database_progress_bar);
```

Comment:
good use of BigOwlActivity to reduce having to set this all the time in every activity/fragment it's needed in